### ISSUE TITLE
grpc 0.13.0 does not install plugins on MinGW64 / MSYS2?

### ISSUE THREAD

**Issue by emmenlau** [grpc/grpc#6073]
It seems the install of plugins is explicitly disabled in Makefile.template:

```
  install-plugins: $(PROTOC_PLUGINS)
  ifeq ($(SYSTEM),MINGW32)
    $(Q) false
  else
    ...
```

Is there a specific reason for this? It seems at least on MSYS2 with MinGW64, the plugins can be installed by disabling this check, and a simple shell execution seems to work (but I did not perform further testing).

**Comment by nicolasnoble** [comment]
Hnng, I don't necessarily remember why the plugin installation got disabled in msys2 / mingw, no. Probably because we were really cautious in building the msys / mingw support initially and didn't want to pollute systems inadvertently.

**Comment by emmenlau** [comment]
I can say that the plugin, once installed, "works for me". I did not do extensive testing, but an application sending and receiving a number of different message types was able to work correctly for at least a few hours. So in spite of further testing, I would say the mileage is pretty ok.

**Comment by soltanmm-google** [comment]
And up goes the version number.

**Comment by emmenlau** [comment]
Wait, is the issue fixed? Or just closed?

**Comment by hsaliak** [comment]
@emmenlau the issue was closed. We have not verified the plugins and are still cautious per the reasons mentioned by @nicolasnoble

**Comment by emmenlau** [comment]
Dear @hsaliak , please reopen. The plugins work on MSYS2 and my mileage is pretty good. While I understand the sentiments from @nicolasnoble , I do not think its helpful successfully compile on MSYS2 and then intentionally fail installing the plugins. Current status is intentionally breaking the build for no apparent reason.
Maybe better to install the plugins unless users actually report issues?

**Comment by jtattermusch** [comment]
Fixed in #9307.
